class.jetpack.php

```diff
@@ -22,6 +22,8 @@
 	Flag for "activating" the plugin on sites where the activation hook never fired (auto-installs)
 */
 
+require_once( JETPACK__PLUGIN_DIR . '_inc/lib/class.media.php' );
```

Comment:
I get `PHP Fatal error:  Uncaught Error: Call to undefined function jetpack_require_lib() in /home/retrofoc/public_html/wp-content/plugins/jetpack/class.jetpack.php:25`
Should I require this lib before?